module/x/gravity/keeper/invariants.go

```diff
@@ -35,33 +34,10 @@ func ModuleBalanceInvariant(k Keeper) sdk.Invariant {
 			expectedBals[v.Denom] = &newInt
 		}
 
```

Comment:
I felt this invariant func was a little too long and not readable at-a-glance so I created two helpers